src/status_im/ui/screens/events.cljs

```diff
@@ -277,3 +278,10 @@
   :update-geolocation
   (fn [db [_ geolocation]]
     (assoc db :geolocation geolocation)))
+
```

Comment:
did you rebase ? this shouldn't be part of the PR

Reply:
Right, I rebased from develop. I'll remove this.